Common/include/containers/CLookUpTable.hpp

```diff
@@ -47,16 +47,19 @@ class CLookUpTable {
   std::string file_name_lut;
   std::string version_lut;
   std::string version_reader;
-  unsigned long n_points;
-  unsigned long n_triangles;
+  unsigned long *n_points;
+  unsigned long *n_triangles;
   unsigned long n_variables;
-  unsigned long n_hull_points;
+  unsigned long *n_hull_points;
+  unsigned long n_table_levels = 1;
+  su2double *z_values_levels;
 
   /*! 
-   * \brief the lower and upper limits of the enthalpy and progress variable.
+   * \brief the lower and upper limits of the z, y and x variable.
```

Comment:
In the combustion/manifold literature, the generalized formulation uses the terms "control variable". 
https://www.researchgate.net/publication/309147831_State-of-the-art_in_premixed_combustion_modeling_using_flamelet_generated_manifolds

Reply:
I replaced name_x and name_y in the CLookUpTable class by name_CV1 and name_CV2. I think it would be nice to keep the LUT class as general as possible so it's also convenient to use for other applications like data-driven fluid models.

Reply:
Matteo also uses it now for NE thermo problems, right? Makes sense to not couple the independent variables to specific names. For the work with Kaiserslautern, the progress variable is actually a mixture fraction and the source term is set to 0 to simulate species mixing with a lookup table.

Reply:
Yes, we are using the tables for NICFD problems in other research with the control variables being density and static energy.